setup.py

```diff
@@ -1,28 +1,20 @@
 import setuptools
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
-
-
-with open("requirements.txt") as f:
-    requirements = f.read().splitlines()
-
-
 setuptools.setup(
-    name="greenplum-python",
-    version="1.0.0-beta1",
-    install_requires=requirements,
+    name="greenplum-python-fork",
+    version="1.0.0-beta4",
```

Comment:
can we find a way to read the `VERSION` file? Or we need a way to extract the version number from this file and to tag automatically by concourse...

Reply:
yes we can add “version”